### ISSUE TITLE
Unable to build this project using info stated in README

### ISSUE THREAD

**Issue by makivlach** [komercka/adaa-example-spring-boot#6]
1. I've filled params in `./web/src/main/resources/application.yml`
2. ran: `mvn clean install`
3. And then `mvn spring-boot:run` fails

When I try to run `mvn spring-boot:run -X` I receive following error: 
![image](https://user-images.githubusercontent.com/24868335/177783379-32936b3a-0e93-49b9-b3e7-512143540b23.png)

```
[DEBUG]   (f) useTestClasspath = false
[DEBUG] -- end configuration --
[INFO] Attaching agents: []
[DEBUG] JVM argument(s): -Xverify:none -XX:TieredStopAtLevel=1
[DEBUG] Classpath for forked process: /home/mivl/GIT/github/adaa-example-spring-boot/target/classes
[DEBUG] Application argument(s): 
[DEBUG] Environment variable(s): 
Error: Could not find or load main class cz.kb.openbanking.adaa.example.springboot.web.AdaaExampleApplication
Caused by: java.lang.ClassNotFoundException: cz.kb.openbanking.adaa.example.springboot.web.AdaaExampleApplication
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Spring Boot example of the ADAA API usage 1.2.0-SNAPSHOT:
[INFO] 
[INFO] Spring Boot example of the ADAA API usage .......... FAILURE [  0.612 s]
```

I'm attempting run the command inside the root of the project: `
![image](https://user-images.githubusercontent.com/24868335/177783858-4e2b12b9-b28b-48f1-96ff-49d06c20427e.png)

Any idea what I've might done wrong?

**Comment by kuchyale** [comment]
Hello, the executable JAR file is located in a `target` folder of `web` module. So, if you want to run `mvn spring-boot:run` from root folder, you should add `-pl web`, like this: `mvn spring-boot:run -pl web`, or try to execute `spring-boot:run` inside `web` folder.

**Comment by kuchyale** [comment]
README example was fixed in #8. @makivlach, thank you for reporting the issue!
